nornir/core/connections.py

```diff
@@ -36,12 +48,12 @@ def open(
         platform: Optional[str],
         extras: Optional[Dict[str, Any]] = None,
         configuration: Optional[Config] = None,
-    ) -> None:
+    ) -> Any:
```

Comment:
If we are going to return something let's alias it to something like `DeviceConnection` or something like that.